Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/PageObjectExtensions.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Threading;
+using Remotion.Web.Development.WebTesting;
+using Remotion.Web.Development.WebTesting.ControlObjects;
+using Remotion.Web.Development.WebTesting.FluentControlSelection;
+using Remotion.Web.Development.WebTesting.WebDriver;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration;
+
+namespace Remotion.ObjectBinding.Web.Development.WebTesting.IntegrationTests
+{
+  internal static class PageObjectExtensions
+  {
+    public static WebButtonControlObject GetValidateButton (this PageObject pageObject, IBrowserConfiguration browserConfiguration)
+    {
+      if (browserConfiguration.IsFirefox()) //TODO RM-7284: Remove this workaround once the underlying problem is resolved.

```

Comment:
I think the comment could be placed on line 13. I guess the whole class will be deleted once the issue is resolved.